aries-site/src/layouts/content/Example/Container.js

```diff
@@ -27,31 +28,71 @@ export const Container = ({
   else if (!plain) height = { min: 'medium' };
   else height = undefined;
 
+  let bestPractice;
+  let background;
+  let label;
+  if (bestPracticeProp) {
+    if (bestPracticeProp.type === 'do') {
+      background = 'brand';
+      label = 'Do';
+    } else if (bestPracticeProp.type === 'dont') {
+      background = 'red';
+      label = "Don't";
+    }
+
+    bestPractice = (
+      <Box gap="xsmall">
+        <Box
+          pad="xsmall"
+          background={background}
+          fill="horizontal"
+          margin={{ top: 'small' }}
+          round="xxsmall"
+        />
+        <Box>
+          <Text color="text-strong" weight="bold" size="large">
+            {label}
+          </Text>
+          <Paragraph size="small" margin="none">
+            {bestPracticeProp.message}
+          </Paragraph>
+        </Box>
+      </Box>
+    );
+  }
+
   return (
-    <Box
-      align={!template && !screenContainer ? 'center' : undefined}
-      background="background-front"
-      direction="row"
-      height={height}
-      justify="center"
-      margin={showResponsiveControls ? { top: 'xsmall' } : undefined}
-      pad={
-        template || screenContainer
-          ? { horizontal: 'large', top: 'large' }
-          : 'large'
-      }
-      round={
-        !horizontalLayout &&
-        (designer || docs || figma || guidance || screenContainer || template)
-          ? { corner: 'top', size: 'small' }
-          : 'small'
-      }
-      {...rest}
-    />
+    <>
+      <Box
+        align={!template && !screenContainer ? 'center' : undefined}
+        background="background-front"
+        direction="row"
+        height={height}
+        justify="center"
+        margin={showResponsiveControls ? { top: 'xsmall' } : undefined}
+        pad={
+          template || screenContainer
+            ? { horizontal: 'large', top: 'large' }
+            : 'large'
+        }
+        round={
+          !horizontalLayout &&
+          (designer || docs || figma || guidance || screenContainer || template)
+            ? { corner: 'top', size: 'small' }
+            : 'small'
```

Comment:
For these smaller boxes, wondering if we can dial back the corners back one notch?

```suggestion
            : 'xsmall'
```

Reply:
I guess this would affect the other containers as well... 

Would we be better off having a specialized DoDontContainer? It could consume Container as an import and pass a round prop in.